.github/workflows/test-windows.yml

```diff
@@ -46,11 +46,22 @@ jobs:
         with:
           token: ${{ secrets.TC_CLOUD_TOKEN }}
 
+      # Forked Pull Requests don't have access to GitHub secrets hence cannot use Test Containers Cloud
+      # this will disable the tests that require access to the Test Containers Cloud.
+      - name: Disable the required tests if needed
+        id: disabled-tests
+        run:
+          if [ "${{github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false}}" = "true" ] ; then
+            echo "list=" >> "$GITHUB_OUTPUT"
+          else
+            echo "list=&FullyQualifiedName!~Elastic.Apm.MongoDb.Tests" >> "$GITHUB_OUTPUT"
```

Comment:
Wanna skip `[xUnit.net 00:11:44.56]     Elastic.Apm.MongoDb.Tests.MongoApmTests.ApmAgent_ShouldCorrectlyCaptureSpan [FAIL]`

since it failed when using docker